benchmarks/plugin-manifest/README.md

```diff
@@ -9,13 +9,13 @@ Benchmarks the current production version of the plugin unless you use `gatsby-d
 
 ```shell
 # In the root of the Gatsby repository
-$ yarn run watch --scope=gatsby-plugin-manifest .
```

Comment:
This denotes a shell input. Not really a "bug" imo.

Reply:
i know - but i tried to unify it, because the most shell commands in docs are without the `$`, and it is more convenient for copy paste ...

Reply:
Thanks for the clarification. Then it makes more sense.